re_data_ui/src/contexts/redataOverviewContext.ts

```diff
@@ -187,4 +190,5 @@ export const RedataOverviewContext = React.createContext<OverviewData>({
   generated_at: '',
   tests: [],
   loading: true,
+  dbtMapping: {},
```

Comment:
not sure what name we can use here. `dbtNodeIdMapping`? @mateuszklimek any suggestions?

Reply:
Hm... I think dbtMapping is good for me :)

Reply:
i'll change it back to dbtMapping